app/ui-message/client/messageBox/notifyUserActivity.html

```diff
@@ -0,0 +1,20 @@
+<template name="notifyUserActivity" args="rid">
+	{{#with data}}
+		<div class="rc-message-box__typing">
+			<span class="rc-message-box__typing-user">{{users}}</span>
+			{{#if multi}}
+				{{#if selfActivity}}
+					{{_ "are_also"}} {{_ action}}
```

Comment:
We can eliminate this case `also`. Let's remove this control of the current user also typing, not useful and we do not support this on mobiles anyways.